docs/source/mturk.rst

```diff
@@ -0,0 +1,181 @@
+..
+  Copyright (c) 2017-present, Facebook, Inc.
+  All rights reserved.
+  This source code is licensed under the BSD-style license found in the
+  LICENSE file in the root directory of this source tree. An additional grant
+  of patent rights can be found in the PATENTS file in the same directory.
+
+Using Mechanical Turk
+===============
+
+In ParlAI, you can use Amazon Mechanical Turk for **data collection**, **training** and **evaluation** of your dialog model. 
+
+Human Turkers are viewed as just another type of agent in ParlAI, and hence person-to-person, person-to-bot, or multiple people and bots in group chat can all talk to each other within the same framework. 
+
+The human Turkers and other agents all communicate in observation/action dict format. During the conversation, the message that human Turkers receive is rendered on the live chat webpage in a pretty printed format, similar to the following:
+
+.. figure:: _static/img/mturk-small.png
+   :align: center
+
+   Example: Human Turker participating in a QA data collection task
+
+General Concepts
+---------------
+
+.. figure:: _static/img/mturk-flowchart.png
+   :width: 400px
+   :align: center
+
+   Diagram for a simple person-to-bot setup *
```

Comment:
nit: can you center and italicize this line? (and the caption for the other image as well?)